package.json

```diff
@@ -37,7 +37,6 @@
     "jsdoc-to-markdown": "^2.0.1",
     "mocha": "^3.2.0",
     "mochainon": "^1.0.0",
-    "wary": "^1.1.0"
```

Comment:
> as the tests didn't explicitly check

There's lots of different image-tests, and the ones I was looking at did verify that the output matched expected - I guess we must have been looking at different tests.

> Temporary files / directories are on the local file system

I was thinking maybe `/tmp` was in a RAM-disk, and the local repository was on a HDD with slow write speeds. But that's such a rare corner-case it's probably not worth worrying about :)